### ISSUE TITLE
Imputation with mi() ignored when variable name contains dot

### ISSUE THREAD

**Issue by jlverissimo** [paul-buerkner/brms#526]
Using mi() with variables whose (column) name contains a dot leads to a multivariate model being fit with the NA rows removed. Is this expected?

```
foo <- data.frame(A=rnorm(10, 10, 10))
foo$B <- foo$A + rnorm(10, 20, 20)
foo$C <- foo$B + rnorm(10, 30, 30)

foo[3,]$C <- NA
foo$C.2 <- foo$C

bform <- bf(A ~ B + mi(C)) + bf(C | mi() ~ B) + set_rescor(FALSE)
bform2 <- bf(A ~ B + mi(C.2)) + bf(C.2 | mi() ~ B) + set_rescor(FALSE)

m1 <- brm(bform, foo)
m2 <- brm(bform2, foo)
```
m2 produces "Warning message: Rows containing NAs were excluded from the model".

**Comment by paul-buerkner** [comment]
This should be fixed in the latest release version (2.5.0). Which version of brms are you using?

**Comment by jlverissimo** [comment]
Perfect, thank you! I mistakenly thought I had upgraded before checking the issue, but looks like I was still at 2.2.0.
